crates/gen/src/method.rs

```diff
@@ -8,6 +8,7 @@ pub struct Method {
     pub params: Vec<Param>,
     pub return_type: Option<Param>,
     pub vtable_offset: u32,
+    pub overload: u32,
```

Comment:
Out of curiousity, why did you choose `u32` instead of `u8` or `u16` for these? Presumably we'll never have more than 256 or certainly not 65536 overloads or virtual methods, no?

Reply:
I just tend to use `u32` for counters. It's not likely there would be anywhere near 256 overloads but there could well be a thousands offsets.